mantle/kola/tests/misc/boot-mirror.go

```diff
@@ -230,6 +240,10 @@ func detachPrimaryBlockDevice(c cluster.TestCluster, m platform.Machine) {
 		}); err != nil {
 			c.Fatalf("Failed to retrieve boot ID: %v", err)
 		}
+
+		// Give some time to the host before doing the reboot to avoid panic.
```

Comment:
I guess we'll always have the commit message/history, but it might be nice to have more context here on why this is needed.